docs/Documentation/Applications/IDAES_Solvers/bonmin.md

```diff
@@ -0,0 +1,5 @@
+# Bonmin
```

Comment:
Is the plan to add more documentation to each of these individual page? Otherwise it doesn't seem like we need the separate pages.

Reply:
I agree.